manifests/vhost.pp

```diff
@@ -263,8 +267,12 @@
 
   # Same as above, but for logroot
   if ! defined(File[$logroot]) {
+    $logroot_ensure_real = $ensure? {
+      present => $logroot_ensure,
+      default => 'absent',
+    }
     file { $logroot:
-      ensure  => $logroot_ensure,
+      ensure  => $logroot_ensure_real,
```

Comment:
I don't like this. The reason we added a `$logroot_ensure` parameter rather than just using `$ensure` is we didn't want to remove logs when a vhost is being removed.